code/headers/test_usart.cpp

```diff
@@ -0,0 +1,41 @@
+#include "test_usart.hpp"
```

Comment:
the .cpp file is located in the headers folder?

Reply:
automated clion stuff sorry about that